lib/osf-components/addon/components/search-result-card/styles.scss

```diff
@@ -64,3 +76,17 @@
     position: relative;
     bottom: 2px;
 }
+
+.cp-panel {
+    padding: 0 10px;
+
+    div > dl {
+        border-top: 2px solid $color-border-gray;
+        padding-top: 10px;
+    }
+
+    dl > dt,
+    dl > dd {
+        margin: 15px 0 0 10px;
+    }
```

Comment:
These can be simplified to
```suggestion
    dl {
        border-top: 2px solid $color-border-gray;
        padding-top: 10px;
    }

    dt,
    dd {
        margin: 15px 0 0 10px;
    }
```

Reply:
I've tested with the format you have and it will not pick up in the div, the div > dl and the like were the ones that worked. I am re-testing and re-building currently to verify.

Reply:
It will work if I halve the padding for definitions and terms:

<img width="1625" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/83dedca4-623a-41ff-b46d-0c7af4be201c">

Reply:
@futa-ikeda this is now updated.